pkg/cloudinfo/services.go

```diff
@@ -38,3 +41,33 @@ func (c *Client) GetServiceRegions(ctx context.Context, cloudProvider string, se
 
 	return regionIds, nil
 }
+
+func (c *Client) PKEImageName(cloudProvider, service, os, kubeVersion, pkeVersion, region string) (string, error) {
+	opts := &cloudinfo.GetImagesOpts{
+		Version:    optional.NewString(kubeVersion),
+		Os:         optional.NewString(os),
+		PkeVersion: optional.NewString(pkeVersion),
+		LatestOnly: optional.NewString("true"),
+	}
+	images, _, err := c.apiClient.ImagesApi.GetImages(context.Background(), cloudProvider, service, region, opts)
+	if err != nil {
+		return "", errors.WrapIfWithDetails(
+			err, "couldn't get PKE images",
+			"cloudProvider", cloudProvider,
+			"service", service,
+			"region", region,
+			"getImagesOpts", opts,
+		)
+	}
+	if len(images) <= 0 {
```

Comment:
```suggestion
	if len(images) == 0 {
```
`len` will never return a negative value.